spartan/aztec-network/values/exp-1.yaml

```diff
@@ -0,0 +1,56 @@
+network:
+  public: false
+
+aztec:
+  slotDuration: 36
+  epochDuration: 32
+  realProofs: false
+
+images:
+  aztec:
+    pullPolicy: Always
+
+telemetry:
+  enabled: true
+  otelCollectorEndpoint: http://35.197.100.168:4318
+
+validator:
+  replicas: 1
+  validatorKeys:
+    - 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+  validatorAddresses:
+    - 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+
+  resources:
+    requests:
+      memory: "512Mi"
+  validator:
+    disabled: false
+
+bootNode:
+  peerIdPrivateKey: 080212200ba8451c6d62b03c4441f0a466c0bce7a3a595f2cf50a055ded3305c77aa3af0
+  validator:
+    disabled: true
+
+proverAgent:
+  replicas: 4
```

Comment:
If `realProofs=false` we don't need high powered provers

Reply:
Good catch, thanks.